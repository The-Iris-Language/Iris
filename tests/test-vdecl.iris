.. test declaration of all types of variables(primitive types and class types)
class Main () {
  public:
    int main() {
      int i;
      bool b;
      char c;
      string s;
      Gecko g;
      Animal a;
      return 0;
    }
}
