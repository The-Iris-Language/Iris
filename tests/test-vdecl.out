class Main of Object () {
public:
int main() {
int i;
bool b;
char c;
string s;
Gecko g;
Animal a;
return 0;
}
}
